.landscaper/ingress-controller/resources.yaml

```diff
@@ -12,17 +12,17 @@ input:
 type: helm.io/chart
 name: ingress-nginx-chart
 relation: external
-version: 4.4.2
+version: 4.5.2
 access:
   type: ociRegistry
-  imageReference: eu.gcr.io/gardener-project/landscaper-service/charts/ingress-nginx@sha256:ef3726917d27503e77b165fa5df8aeede1657c179a165504ab8fbef86c0b67ce
+  imageReference: eu.gcr.io/gardener-project/landscaper-service/charts/ingress-nginx@sha256:da4623068792ce13685f6d0548966a04b1754886275445fdc57b56d42a607d03
 ...
 ---
 type: ociImage
 name: ingress-nginx-controller
-version: v1.5.1
+version: v1.6.4
 relation: external
 access:
   type: ociRegistry
-  imageReference: eu.gcr.io/gardener-project/landscaper-service/nginx-ingress/controller@sha256:97fa1ff828554ff4ee1b0416e54ae2238b27d1faa6d314d5a94a92f1f99cf767
+  imageReference: eu.gcr.io/gardener-project/landscaper-service/nginx-ingress/controller@sha256:e727015a639975f4fc0808b91f9e88a83c60938b640ee6c2f5606ddd779c858d
```

Comment:
When I look at eu.gcr.io/gardener-project/landscaper-service/nginx-ingress/controller I could only find one image with the old hash 97fa... but no one with the new hash e7270...

Reply:
@achimweigel I just checked, and this revision is definately available: 
https://console.cloud.google.com/gcr/images/gardener-project/eu/landscaper-service/ingress-nginx/controller@sha256:e727015a639975f4fc0808b91f9e88a83c60938b640ee6c2f5606ddd779c858d/details